elastic/kibana
# [Mappings editor] Clean up dynamic_templates code

Addresses this comment: https://github.com/elastic/kibana/pull/54523#discussion_r365649028

### How to review
- Confirm comment above is addressed as expected
- Confirm dynamic_templates functionality still works as expected
